authors/apps/authentication/templates/email_verification.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Authors Haven</title>
+    <style>
```

Comment:
Why is the css not separated from the html?

Reply:
It is a good practice to separate the two. For the purpose of this sprint, we settled as a team to ensure the endpoint feature is working for demonstration purposes rather than the front end. However, we agreed to work on the separation of CSS and HTML when we come to the handling of the frontend. For this reason, we left it as is. @Collins33